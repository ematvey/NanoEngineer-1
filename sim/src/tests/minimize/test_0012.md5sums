stdout FD02724A82513C290C46018B77D18DF9
stderr D41D8CD98F00B204E9800998ECF8427E
test_0012.xyz 37627A982FAA366736C101B742556F2A
