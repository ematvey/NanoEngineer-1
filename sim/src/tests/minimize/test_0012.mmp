mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (0.907421, 0.292459, 0.188762, -0.235425) (6.147814) (0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (123)
mol (Chunk-C501) tub
atom 1 (6) (38, 22, 0) def
atom 2 (0) (-406, 467, -444) def
bond1 1
atom 3 (0) (482, -421, -444) def
bond1 1
atom 4 (6) (927, 911, 889) def
bond1 1
atom 5 (0) (747, 1637, 705) def
bond1 4
atom 6 (0) (1678, 795, 768) def
bond1 4
atom 7 (6) (-851, -866, 889) def
bond1 1
atom 8 (0) (-714, -1607, 729) def
bond1 7
atom 9 (0) (-1591, -706, 752) def
bond1 7
atom 10 (6) (697, 609, 2373) def
bond1 4
atom 11 (0) (478, 1260, 2720) def
bond1 10
atom 12 (0) (1345, 350, 2698) def
bond1 10
atom 13 (6) (-477, -535, 2431) def
bond1 10 7
atom 14 (0) (-1116, -289, 2782) def
bond1 13
atom 15 (0) (-183, -1130, 2821) def
bond1 13
egroup (123)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part 123
