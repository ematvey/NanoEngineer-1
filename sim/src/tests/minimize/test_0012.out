======= test_0012.test =======
======= exitvalue =======
0
======= stderr =======
cutover to fine tolerance at 10
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0012.mmp  > test_0012.xyz
final RMS gradient=1015.467017 after 13 iterations
======= test_0012.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Date and Time: Mon Nov 21 18:46:02 2005
# Input File:test_0012.mmp
# Output File: test_0012.xyz
# Trace File: test_0012.trc
# Number of Atoms: 15
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
#
# 0 columns:
#
0 3550.60 4227.89
1 2624.64 3592.73
2 2222.26 3154.62
3 1958.94 2842.14
4 1733.53 2549.57
5 1497.79 2235.02
6 1192.49 1837.07
7 740.52 1132.36
8 910.55 1161.42
9 1016.14 1275.91
10 1015.46 1275.19
11 1015.47 1275.20
12 1015.47 1275.20
13 1015.47 1275.20
# Done: Minimization final rms: 1015.467017, highForce: 1275.198815
======= test_0012.xyz =======
15
RMS=1015.467017
C 0.042598 0.017086 -0.067141
X -0.418301 0.463142 -0.518624
X 0.495400 -0.443953 -0.516759
C 0.978131 0.965869 0.858575
X 0.809129 1.722942 0.733617
X 1.731489 0.825684 0.685271
C -0.871709 -0.895041 0.908811
X -0.731729 -1.658919 0.776134
X -1.630933 -0.736406 0.788833
C 0.678583 0.608268 2.409941
X 0.426433 1.249819 2.795389
X 1.331438 0.334392 2.751416
C -0.475507 -0.529987 2.437090
X -1.101272 -0.247443 2.828362
X -0.209750 -1.178454 2.798086
